src/main/java/frc/robot/RobotContainer.java

```diff
@@ -285,4 +285,8 @@ private void configureBindings() {
   public Command getAutonomousCommand() {
     return autoChooser.getSelected();
   }
+
+  public static void setDriveMode(String driveMode) {
+    SmartDashboard.putString("Drive mode", driveMode);
+  }
```

Comment:
You're overcomplicating things. Get rid of this method entirely, just directly call SmartDashboard.putString() from the initialize() methods.